partnerapp/src/InteractionPreviewView.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { FC } from "react";
+import { observer } from "mobx-react-lite";
+import { useParams, useHistory } from "react-router-dom";
+import { CampaignStore } from "./campaign-store";
+import { Button } from "reactstrap";
+
+export const InteractionPreviewView: FC<{
+  store: CampaignStore;
+}> = observer(({ store }) => {
+  const history = useHistory();
+  const params: { campaignId: string } = useParams();
+  const campaign = store.getCampaign(params.campaignId);
+
+  if (!campaign) {
+    console.log("Could not find campaign!", params.campaignId);
```

Comment:
To improve debugging you might want to use console.error here.
```suggestion
    console.error("Could not find campaign!", params.campaignId);
```